test/test_flow_control_window.py

```diff
@@ -52,6 +52,28 @@ def test_flow_control_decreases_with_sent_data(self):
         remaining_length = self.DEFAULT_FLOW_WINDOW - len(b'some data')
         assert (c.local_flow_control_window(1) == remaining_length)
 
+    def test_flow_control_decreases_with_sent_data_with_padding(self):
+        """
+        When data is sent on a stream, the flow control window should drop.
+        """
+        c = h2.connection.H2Connection()
+        c.send_headers(1, self.example_request_headers)
+
+        c.send_data(1, b'some data', pad_length=5)
+        remaining_length = self.DEFAULT_FLOW_WINDOW - len(b'some data') - 6
+        assert (c.local_flow_control_window(1) == remaining_length)
+
+    def test_flow_control_decreases_with_sent_data_zero_length_padding(self):
+        """
+        When data is sent on a stream, the flow control window should drop.
```

Comment:
Same note here: this docstring needs to be updated to note that we're testing that with zero-length padding the flow control window should drop by one byte more than the data we sent.